platform/00-kubernetes-minikube-setup.sh

```diff
@@ -55,13 +55,14 @@ echo -e "${C_GREEN}Installing packages if needed...${C_RESET_ALL}"
 case "${PLATFORM}" in
 
   Darwin)
-    minikube version || brew tap-new "$USER"/local-minikube; brew extract --version=1.26.1 minikube "$USER"/local-minikube; brew install minikube@1.26.1
+    minikube version || brew tap-new "$USER"/local-minikube; brew extract --force --version=1.26.1 minikube "$USER"/local-minikube; brew install minikube@1.26.1
```

Comment:
#358 Should resolve the issue with this line

Reply:
With #358 merged, this needs a rebase.

Reply:
thanks for pointing this out...just rebased and only added crane install